tool/tctl/common/bots_command.go

```diff
@@ -55,26 +56,33 @@ type BotsCommand struct {
 	lockExpires string
 	lockTTL     time.Duration
 
-	botName  string
-	botRoles string
-	tokenID  string
-	tokenTTL time.Duration
-	addRoles string
+	botName    string
+	botRoles   string
+	tokenID    string
+	tokenTTL   time.Duration
+	addRoles   string
+	instanceID string
 
 	allowedLogins []string
 	addLogins     string
 	setLogins     string
 
-	botsList   *kingpin.CmdClause
-	botsAdd    *kingpin.CmdClause
-	botsRemove *kingpin.CmdClause
-	botsLock   *kingpin.CmdClause
-	botsUpdate *kingpin.CmdClause
+	botsList         *kingpin.CmdClause
+	botsAdd          *kingpin.CmdClause
+	botsRemove       *kingpin.CmdClause
+	botsLock         *kingpin.CmdClause
+	botsUpdate       *kingpin.CmdClause
+	botsInstance     *kingpin.CmdClause
+	botsInstanceShow *kingpin.CmdClause
+	botsInstanceList *kingpin.CmdClause
+	botsInstanceAdd  *kingpin.CmdClause
+
+	stdout io.Writer
 }
 
 // Initialize sets up the "tctl bots" command.
 func (c *BotsCommand) Initialize(app *kingpin.Application, config *servicecfg.Config) {
-	bots := app.Command("bots", "Operate on certificate renewal bots registered with the cluster.")
+	bots := app.Command("bots", "Operate on certificate renewal bots registered with the cluster.").Alias("bot")
```

Comment:
Whilst we're here, could we update this doc string ? Perhaps something like "Manage Machine ID Bots" ? I feel like "certificate renewal bots" is a old phraseology we don't tend to reference much.